presto-product-tests/src/main/resources/tempto-configuration.yaml

```diff
@@ -32,6 +32,24 @@ databases:
     jdbc_password: na
     jdbc_pooling: false
 
+  alice@presto:
+    host: localhost
+    server_address: ${databases.presto.host}:8080
+    jdbc_driver_class: ${databases.presto.jdbc_driver_class}
+    jdbc_url: jdbc:presto://${databases.presto.server_address}/hive/${databases.hive.schema}
+    jdbc_user: alice
+    jdbc_password: na
+    jdbc_pooling: false
+
+  bob@presto:
+    host: localhost
+    server_address: ${databases.presto.host}:8080
+    jdbc_driver_class: ${databases.presto.jdbc_driver_class}
+    jdbc_url: jdbc:presto://${databases.presto.server_address}/hive/${databases.hive.schema}
+    jdbc_user: bob
+    jdbc_password: na
+    jdbc_pooling: false
+
```

Comment:
I add these definitions in here too, since while running the test in Intellij, this file was being used. My understanding is that `presto-product-tests/src/main/resources/tempto-configuration.yaml` is being used for running the test in Intellij and `presto-product-tests/conf/tempto/tempto-configuration-for-docker-default.yaml` is being used for running the test from docker. Is this correct?